code/game/turfs/simulated/outdoors/dirt.dm

```diff
@@ -6,3 +6,107 @@
 	turf_layers = list(/turf/simulated/floor/outdoors/rocks)
 	initial_flooring = /decl/flooring/dirt
 	can_dig = TRUE
+
+/turf/simulated/floor/outdoors/newdirt
+	name = "dirt"
+	desc = "Looks dirty."
+	icon = 'icons/turf/outdoors.dmi'
+	icon_state = "dirt0"
+	edge_blending_priority = 2
+	initial_flooring = /decl/flooring/outdoors/newdirt
+	can_dig = TRUE
+
+/decl/flooring/outdoors/newdirt
+	name = "dirt"
+	desc = "Looks dirty."
+	icon = 'icons/turf/outdoors.dmi'
+	icon_base = "dirt0"
+	footstep_sounds = list("human" = list(
+		'sound/effects/footstep/asteroid1.ogg',
+		'sound/effects/footstep/asteroid2.ogg',
+		'sound/effects/footstep/asteroid3.ogg',
+		'sound/effects/footstep/asteroid4.ogg',
+		'sound/effects/footstep/asteroid5.ogg',
+		'sound/effects/footstep/MedDirt1.ogg',
+		'sound/effects/footstep/MedDirt2.ogg',
+		'sound/effects/footstep/MedDirt3.ogg',
+		'sound/effects/footstep/MedDirt4.ogg'))
+
+/turf/simulated/floor/outdoors/newdirt/Initialize(mapload)
+	var/possibledirts = list(
```

Comment:
Please make this a global var outside of Initialize(), or a static var on the turf Declaring it and discarding it every time one of these turfs init is expensive. Same for the other instances below.